src/main/java/com/careerfair/q/controller/student/StudentController.java

```diff
@@ -28,10 +26,10 @@
     /**
      *
      * @param id
-     * @param updateStudentRequest
-     * @return
+     * @param updatedValues
+     * @return UpdateStudentResponse
      */
-    UpdateStudentResponse updateStudent(String id, UpdateStudentRequest updateStudentRequest);
+    UpdateStudentResponse updateStudent(String id, Map<String, Object> updatedValues);
 
```

Comment:
Make a UpdateStudentRequest object. You should then figure out the fields that may be passed by the front end to you instead of having a map as a request body.